config/files/server/usr/etc/ssh/sshd_config.d/99-hardening.conf

```diff
@@ -20,4 +26,32 @@ TCPKeepAlive no
 ClientAliveCountMax 2
 
 # Forbid sshing as root
-PermitRootLogin no
\ No newline at end of file
+PermitRootLogin no
+
+# Enable Pubkey Authentification
+PubkeyAuthentication yes
+
+# Block empty passwords
+PermitEmptyPasswords no
+
+# Allow only 256bit AES encryption
```

Comment:
why is this needed?

Reply:
default should be

```
# Ciphers aes128-ctr,aes192-ctr,aes256-ctr
# MACs hmac-sha2-256,hmac-sha2-512
```

so this only accepts the most secure ones. I read somewhere that modern ssh clients default to 256 anyways.

Reply:
256 isn't "more secure" per se

https://security.stackexchange.com/posts/19762/revisions

128 is fine.

Reply:
Please remove the cipher restrictions and mac restrictions